gossip/src/cluster_info.rs

```diff
@@ -129,6 +129,8 @@ pub const MAX_INCREMENTAL_SNAPSHOT_HASHES: usize = 25;
 /// Maximum number of origin nodes that a PruneData may contain, such that the
 /// serialized size of the PruneMessage stays below PACKET_DATA_SIZE.
 const MAX_PRUNE_DATA_NODES: usize = 32;
+/// Prune data prefix for PruneMessage
+const PRUNE_DATA_PREFIX: &[u8] = "SOLANA_PRUNE_DATA".as_bytes();
```

Comment:
this would assume we never go to the trouble of extending the MTU. there's a [similar treatment](https://github.com/solana-labs/solana/blob/master/docs/src/proposals/off-chain-message-signing.md#signing-domain-specifier) specified for off-chain message signing. afaik the [associated runtime support](https://github.com/solana-labs/solana/blob/master/docs/src/proposals/off-chain-message-signing.md#runtime-considerations) hasn't been added yet, but we should probably use the same value for the sake of future proofing